docs/Contributors_Guide/github_workflow.rst

```diff
@@ -23,6 +23,140 @@ then the develop branch is used to create a main_vx.y release of METplus,
+
+   - Click on the three dots to the right of the project name to see **More options**
+     and select **Workflows**.  Modify these settings as follows.
+
+      - Enable the **Item added to project** workflow and set the status to **Backlog**.
+
+      - Enable the **Item reopened** workflow and set the status to **In progress**.
+
+      - Enable the **Item closed** workflow and set the status to **Done**.
+
+      - Enable the **Pull request merged** workflow and set the status to **Done**.
+
+      - Leave all other workflows disabled.
+
+   - Click the backward arrow to return to the project page.
+
+4. Create project **Views**.
+
+   - Each view appears as a tab on the project page.  Create a new view as described below.
+
+      - Select the **+ New view** option.
+
+      - Click on the view name to modify it, rename it as **All Cycles**, and hit **Enter**.
+
+      - Click on the downward arrow and, under **Configuration**, select the **Fields** option. Enable
+        the options for **Title**, **Repository**, **Assignees**, **Cycle**, **Status**,
+        **Linked Pull Request**, and **Reviewers**. Drag and drop the items to reorder them as listed above.
+
+      - In the resulting view, click the three dots in the **Status** column. Select the **Sort descending**
+        and **Group by values** options.
+
+      - The blue dot on the downward error for this tab indicates that there are unsaved changes.
+        Select the **Save changes** option.
+
+   - Click on the downward arrow and select **Duplicate View**. Do this 7 times and name/refine these views as follows.
+
+      - View name **All Required** shows all items labelled as *REQUIRED* for the development or official release.
+        Click on the 3 horizontal bars and define the filtering criteria as
+        **label:"required: FOR OFFICIAL RELEASE","required: FOR DEVELOPMENT RELEASE" is:open**.  Click **Save Changes**.
+
+      - View name **Beta1** to **Beta5** shows items for each individual development cycle.
+        Click on the 3 horizontal bars and define the filtering criteria as **cycle:Beta1**, **cycle:Beta2**,
+        and so on.  Click **Save Changes**.
+
+      - View name **Closed Issues** shows issues that have been closed across all development cycles.
+        Click on the 3 horizontal bars and define the filtering criteria as **is:closed is:issue**.
+        Click on the 3 dots in the **Cycle** column and select **Group by values**.  Click **Save Changes**.
+
+   - Delete any other views created by default by clicking the downward arrow next to the view name and
+     selecting **Delete view**.
+
+5. Refine the project settings, development cycle dates, and views, as needed, based on the preferences
+   of the development team.
+
+6. Link the new project to each repository.
+
+   - Navigate to the project page for each repository managed by this project
+     (e.g. `METplus Projects <https://github.com/dtcenter/METviewer/projects>`_).
```

Comment:
This says "METplus Projects", but links to METviewer projects.  I am guessing that wasn't intended, but I'm not sure if you'd rather say "METviewer projects" or link to METplus or if you have no preference.  Please fix however you'd like.